src/lib.rs

```diff
@@ -1381,6 +1381,32 @@ impl<T, E> UnwrapThrowExt<T> for Result<T, E>
 where
     E: core::fmt::Debug,
 {
+    #[cfg_attr(debug_assertions, track_caller)]
+    fn unwrap_throw(self) -> T {
+        if cfg!(all(
+            target_arch = "wasm32",
+            not(any(target_os = "emscripten", target_os = "wasi"))
+        )) {
```

Comment:
This needs to be gated behind `cfg(debug_assertions)`, the same as the default `unwrap_throw` implementation; see https://github.com/rustwasm/wasm-bindgen/pull/2995#pullrequestreview-1041858759.

(The `cfg(feature = "std")` might not be needed anymore after #4005.)

Reply:
Thank you!
Addressed in #4042.